src/Microsoft.ServiceFabric.Services/Client/ServicePartitionResolver.cs

```diff
@@ -637,11 +639,16 @@ public static ServicePartitionResolver GetDefault()
 
                 // wait before retry
                 await Task.Delay(
-                    new TimeSpan((long)(Rand.NextDouble() * maxRetryInterval.Ticks)),
+                    Utility.GetRetryDelay(this.GetRetryJitter(maxRetryInterval), currentRetryCount++),
                     cancellationToken);
             }
         }
 
+        private TimeSpan GetRetryJitter(TimeSpan maxRetryInterval)

```

Comment:
This uses the max retryinterval as jitter.

Reply:
Reverting this to what we had it before